Dockerfile

```diff
@@ -1,19 +1,17 @@
 # Build Stage
 FROM rust:1.72.1 as build
-COPY . /node/
+COPY /node/ Makefile /node/
```

Comment:
do I understand correctly that this way Makefile will be executed from a different location from within docker and locally? Please avoid that either by copying Makefile to the root, or by moving it to node/ directory.

Reply:
Yeah, I fixed it to be in the root of the container, just like on the local machine.